api/src/main/kotlin/pl/edu/pk/backend/controller/TripController.kt

```diff
@@ -26,20 +28,30 @@ class TripController(private val tripService: TripService) {
       return
     }
     val body = ctx.safeBodyAsJson() ?: return
-    val cost = body.getString("cost", "")
+    val iCost = body.getInteger("cost", 1)
```

Comment:
`cost` is okay for the name, just make `patchTrip` accept int (didn't check but should be possible)